js/components/textTrackPanel.js

```diff
@@ -127,8 +129,8 @@ var TextTrackPanel = React.createClass({
                   this.props.closedCaptionOptions.textEnhancement
                 )
               }
-              >
-              {this.props.cueText}
+             >
+             <span dangerouslySetInnerHTML={Utils.createMarkup(this.props.cueText)}></span>
```

Comment:
This is a question to @mistermendez, he is going for it, we use this pattern in a couple other places